src/useFirebaseUser.js

```diff
@@ -3,13 +3,13 @@ import firebase from 'firebase/app'
 import 'firebase/auth'
 import { getConfig } from 'src/config'
 import createAuthUser from 'src/createAuthUser'
+import logDebug from 'src/logDebug'
 
-const defaultTokenChangedHandler = async (authUser) => {
+const defaultTokenChangedHandler = async (authUser, userToken) => {
```

Comment:
Why does this need to also receive an ID token when it already has access via AuthUser?

Reply:
So I can't tell you for sure except that when I removed it and used the get getToken() function on authUser to get the token, I ended up with a number of JWT errors. I'm not sure if this is because the token was undefined (i.e. could authuser be initialized without the right gettoken function?) or if it was because it was malformed. But sending the token that is returned from the getIdTokenResult call works perfectly. Too fried to try and figure it out.

Reply:
Nope, you were right. The reason it was erring for me was because I had a custom tokenhandler in my test app that I forgot to modify. Should be all good now!